deepchem/models/torch_models/dmpnn.py

```diff
@@ -7,11 +7,41 @@
 from deepchem.models.torch_models import layers
 from deepchem.models.torch_models import TorchModel
 
+try:
+  from torch_geometric.data import Data, Batch
+except ModuleNotFoundError:
+  raise ModuleNotFoundError(
+      "This model requires PyTorch Geometric to be installed.")
+
```

Comment:
Exceptions in `dmpnn.py` is not needed. Catch all exceptions in `models/__init__.py`. Let the module dmpnn be as it is.

Reply:
ok